base_layer/core/tests/chain_storage_tests/chain_backend.rs

```diff
@@ -109,7 +109,10 @@ fn lmdb_insert_contains_delete_and_fetch_header() {
 
     // Cleanup test data - in Windows the LMBD `set_mapsize` sets file size equals to map size; Linux use sparse files
     if std::path::Path::new(&temp_path).exists() {
-        std::fs::remove_dir_all(&temp_path).unwrap();
+        match std::fs::remove_dir_all(&temp_path) {
```

Comment:
These should probably rather use `tempfile::tempdir()` which cleans up the temporary file on drop. Any reason why these changed?

Reply:
Yes, this is a funny one, as something changed recently on my one Windows computer that changed the default behaviour for a while (but not on the other). For some reason, this call did not remove the db folder as intended, as it seems the handles were not released properly. The `.unwrap()` in `std::fs::remove_dir_all(&temp_path).unwrap();` caused a runtime exception, which in turn caused the unit tests to fail. However, it was possible to delete the test folder afterwards with File Explorer.